addons/web/static/src/legacy/legacy_setup.js

```diff
@@ -8,6 +8,7 @@ import {
     makeLegacyDialogMappingService,
     makeLegacyCrashManagerService,
     makeLegacyCommandService,
+    makeLegacyDropdownService,
```

Comment:
[IMP] web: deploy ui and hotkey as legacy services too

maybe explain why we need this, that it isn't complete (no all new services are there, only the two necessary for the dropdowns)